tests/farm/LiquidityMiningMigration.js

```diff
@@ -745,6 +770,323 @@ describe("LiquidityMiningMigration", () => {
+			expect(await loanToken.totalSupply()).bignumber.equal(depositAmount);
+		});
+	});
+
+	describe("Test WRBTC lending with liquidity mining", () => {
+		it("Should lend to the pool at the liquidity mining V1, migrate it and lend to liquidity mining V2", async () => {
+			const depositAmount = new BN(wei("400", "ether"));
+			await liquidityMining.add(loanTokenWRBTC.address, new BN(10), true);
+			await loanTokenWRBTC.setLiquidityMiningAddress(liquidityMining.address);
+
+			const tx = await loanTokenWRBTC.mintWithBTC(lender, true, { value: depositAmount });
+
+			await liquidityMining.addAdmin(migrator.address);
+			await liquidityMining.startMigrationGracePeriod();
+			await liquidityMiningV2.addAdmin(migrator.address);
+			await migrator.migratePools();
+			await migrator.migrateUsers([lender]);
+			await migrator.finishUsersMigration();
+			await migrator.migrateFunds();
+
+			await loanTokenWRBTC.setLiquidityMiningAddress(liquidityMiningV2.address);
+			const userInfo = await liquidityMiningV2.getUserInfo(loanTokenWRBTC.address, lender);
+			//expected: user pool token balance is 0, but balance of LM contract increased
+			expect(await loanTokenWRBTC.balanceOf(lender)).bignumber.equal("0");
+			expect(userInfo.amount).bignumber.equal(depositAmount);
+			expect(await loanTokenWRBTC.totalSupply()).bignumber.equal(depositAmount);
+			//expect the Mint event to mention the lender
+			expectEvent(tx, "Mint", {
+				minter: lender,
+				tokenAmount: depositAmount,
+				assetAmount: depositAmount,
+			});
+		});
+
+		it("Should lend to the pool at the liquidity mining V1, migrate it and lend to liquidity mining V2", async () => {
+			const depositAmount = new BN(wei("400", "ether"));
+			await liquidityMining.add(loanTokenWRBTC.address, new BN(10), true);
+			await loanTokenWRBTC.setLiquidityMiningAddress(liquidityMining.address);
+
+			await loanTokenWRBTC.mintWithBTC(lender, true, { value: depositAmount });
+
+			await liquidityMining.addAdmin(migrator.address);
+			await liquidityMining.startMigrationGracePeriod();
+			await liquidityMiningV2.addAdmin(migrator.address);
+			await migrator.migratePools();
+			await migrator.migrateUsers([lender]);
+			await migrator.finishUsersMigration();
+			await migrator.migrateFunds();
+
+			await loanTokenWRBTC.setLiquidityMiningAddress(liquidityMiningV2.address);
+			await loanTokenWRBTC.mintWithBTC(lender, true, { value: depositAmount });
+			const userInfo = await liquidityMiningV2.getUserInfo(loanTokenWRBTC.address, lender);
+			//expected: user pool token balance is 0, but balance of LM contract increased
+			expect(await loanTokenWRBTC.balanceOf(lender)).bignumber.equal("0");
+			expect(userInfo.amount).bignumber.equal(depositAmount.mul(new BN(2)));
+			expect(await loanTokenWRBTC.totalSupply()).bignumber.equal(depositAmount.mul(new BN(2)));
+		});
+
+		it("Should lend to the pool without depositing the pool tokens at the liquidity mining contract", async () => {
```

Comment:
Is this test using any of the features provided by the `LiquidityMining` contracts?